### PR TITLE
Updated agent page

### PR DESCRIPTION
**Pull request by russmiles**
Contributes to #179

### CODE REVIEW

File: docs/user-guide/spring-agent.md [hunk 1]
```diff
@@ -1,56 +1,174 @@
...
 
-Add the Spring Agent to a project by running `@atomist add spring agent`. Ideally you would run this command from a Slack channel that is linked to GitHub repository via `@atomist repo <repo owner name> <repo name>`. 
+Once you're sure you're in a channel attached to the repository that contains your Maven-based Spring Boot application you can ask Atomist to add the agent:
+
+```shell
+@atomist add spring agent
+```
 
-### Manually
+This will result in the bot creating a branch on your repository and making changes to the `pom.xml` and the `application.yml`.
```
Comment (cdupuis) [inline docs/user-guide/spring-agent.md]: ... branch and PR ...

File: docs/user-guide/spring-agent.md [hunk 2]
```diff
@@ -1,56 +1,174 @@
...
+> mvn spring-boot:run
+```
+
+If everything is configured correctly then you should then begin to see application instance lifecycle messages for your Spring Boot application in the corresponding channel for your project:
+
+![Spring Application Lifecycle Messages](/images/agent-messages.png)
+
+## What information is shared by the Spring Agent?
+
+Once your application is running with the agent added then you in the `DEBUG` logs for your application you will see the information that is being sent to Atomist:
```
Comment (cdupuis) [inline docs/user-guide/spring-agent.md]: That sentence reads oddly.

Comment (russmiles) [inline docs/user-guide/spring-agent.md]: Making the change

File: docs/user-guide/spring-agent.md (hunk 1 above)
Comment (russmiles) [inline docs/user-guide/spring-agent.md]: Absolutely, making that change now

**Comment by russmiles** [comment]
Incorporated your amendments @cdupuis

**Pull request merged**